php/example_code/dynamodb/Readme.md

```diff
@@ -70,6 +86,32 @@ following:
 ..\..\vendor\bin\phpunit DynamoDBBasicsTests.php
 ```
 
+**Getting started with DynamoDB and PartiQL**
+
+This scenario shows you how to create an Amazon DynamoDB table for storing movie data. The scenario loads movies into
```

Comment:
This is too extensive. According to the service level README template, the scenarios are supposed to be a bulleted list that just gives the link to the code. See this example for Python:

https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/python/example_code/dynamodb